__tests__/app.test.js

```diff
@@ -69,3 +69,31 @@ describe("GET /api/articles/:article_id", () => {
     })
   })
 })
+
+describe("GET /api/articles", () => {
+  test("200: Responds with an array of articles to the client", () => {
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then((response) => {
+      response.body.articles.forEach((article) => {
```

Comment:
You need to check the length of the array first. If you use a forEach on an empty array, it will not run so your assertions will be skipped over.